src/main/java/com/bookstore/contoller/BookController.java

```diff
@@ -0,0 +1,31 @@
+package com.bookstore.contoller;
+
+import com.bookstore.DTO.BookDto;
+import com.bookstore.DTO.CreateBookRequestDto;
+import com.bookstore.service.BookService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/books")
```

Comment:
add `server.servlet.context-path=/api` to application.properties
```suggestion
@RequestMapping("/books")
```